vital/types/rotation.cxx

```diff
@@ -50,22 +50,20 @@ template < typename T >
 rotation_< T >
 ::rotation_( const T& yaw, const T& pitch, const T& roll )
 {
-  using std::cos;
-  using std::sin;
-  static const double root_two = std::sqrt( static_cast<double>(2.0) );
-  static const T inv_root_two = static_cast< T > ( 1.0 / root_two );
-
-  // compute the rotation from North-East-Down (NED) coordinates to
-  // East-North-Up coordinates (ENU). It is a 180 degree rotation about
-  // the axis [1/sqrt(2), 1/sqrt(2), 0]
-  const rotation_< T > Rned2enu( Eigen::Quaternion< T > ( 0, inv_root_two, inv_root_two, 0 ) );
-  const double half_x = 0.5 * static_cast< double > ( -roll );
-  const double half_y = 0.5 * static_cast< double > ( -pitch );
-  const double half_z = 0.5 * static_cast< double > ( -yaw );
-  rotation_< T > Rx( Eigen::Quaternion< T > ( T( cos( half_x ) ), T( sin( half_x ) ), 0, 0 ) );
-  rotation_< T > Ry( Eigen::Quaternion< T > ( T( cos( half_y ) ), 0, T( sin( half_y ) ), 0 ) );
-  rotation_< T > Rz( Eigen::Quaternion< T > ( T( cos( half_z ) ), 0, 0, T( sin( half_z ) ) ) );
-  *this = Rx * Ry * Rz * Rned2enu;
+  T const half_x = static_cast< T >( 0.5 ) * roll;
+  T const half_y = static_cast< T >( 0.5 ) * pitch;
+  T const half_z = static_cast< T >( 0.5 ) * yaw;
+  T const sin_x = std::sin( half_x );
+  T const cos_x = std::cos( half_x );
+  T const sin_y = std::sin( half_y );
+  T const cos_y = std::cos( half_y );
+  T const sin_z = std::sin( half_z );
+  T const cos_z = std::cos( half_z );
+  *this = Eigen::Quaternion< T >{
```

Comment:
If you could point to your references for these calculations that would be helpful. But I'm willing to trust these are correct and just merge it.

Reply:
I think pointing to a reference in a comment would be useful anyway, since this is a bit challenging to parse.

Reply:
Done